website/docs/reference/global-configs.md

```diff
@@ -405,7 +405,50 @@ Turning on the `WARN_ERROR` config will convert dbt warnings into errors. Any ti
 $ dbt --warn-error run
 ...
 ```
+</File>
+
```

Comment:
Ah, sorry, one last thing - because this feature is new in v1.4, let's just wrap everything below in a ["VersionBlock"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content):
```
<VersionBlock firstVersion="1.4">
... everything you're adding below ...
</VersionBlock>
```

That will work once v1.4 is added to the set of site-wide ["dbt versions,"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/dbt-versions.js#L1) which Matt is tackling in https://github.com/dbt-labs/docs.getdbt.com/pull/2534.

Unfortunately, the `VersionBlock` rendering doesn't yet work _within_ code blocks, so no need to do this for the small change to `profiles.yml.md`.

Reply:
cooooool!